drivers/tja1042/tja1042.c

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2016  OTA keys S.A.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tja1042 trx
```

Comment:
should be `@ingroup drivers_tja1042` only

Reply:
though also these files are not parsed by doxygen, anyway.

Reply:
addressed